logs/charts/Chart.yaml

```diff
@@ -2,9 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 apiVersion: v2
-appVersion: v0.121.0
+appVersion: v0.122.0
```

Comment:
is this used ?

Reply:
Not sure, it was added here #246 and here I am bumping it to stay consistent.

Reply:
Decided to put it as the v0.124.1 which is consistent with the docker image used, see https://github.com/cloudoperators/greenhouse-extensions/commit/469df1a305ba96f8290b37766eefe343fdc589d9